scene/resources/navigation_mesh.cpp

```diff
@@ -46,6 +47,7 @@ void NavigationMesh::create_from_mesh(const Ref<Mesh> &p_mesh) {
 		Vector<Vector3> varr = arr[Mesh::ARRAY_VERTEX];
 		Vector<int> iarr = arr[Mesh::ARRAY_INDEX];
 		if (varr.size() == 0 || iarr.size() == 0) {
+			WARN_PRINT("A mesh surface was skipped when creating a NavigationMesh due to an empty vertex or index array");
```

Comment:
You missed the dot in my previous suggestion:
```suggestion
			WARN_PRINT("A mesh surface was skipped when creating a NavigationMesh due to an empty vertex or index array.");
```

Reply:
damn